xarray_schema/core.py

```diff
@@ -72,6 +72,11 @@ def validate(self, da: xr.DataArray) -> xr.DataArray:
         ------
         SchemaError
         '''
+        if not isinstance(da, xr.DataArray):
+            raise ValueError('Input must be a xarray.DataArray')
+        
+        if da.chunks:
+            assert _check_regular_chunks(da.chunks), 'Good gracious no! Chunks are not regular!'
```

Comment:
irregular chunks are "fine" in some cases. So I don't think we want to raise an error here. I'd rather catch this down in the `chunks` section if regular chunks are specified.
```suggestion
```